src/readbin/func_start.mli

```diff
@@ -0,0 +1,8 @@
+open Bap.Std
```

Comment:
Please, use comments. In general usually you must start with writing and commenting your interface, not vice verse. Usually, when you write a comment you formalize your function, and, as a result, reduces the probability of unreasonable interfaces, like this one for example. The general rule, if you don't know what your function is doing, or can't explain it in a plain English, then you're doing something wrong. 

For example: `Func_start.user something` what it is supposed to do?  What `something` is? Why the function is noun? Nouns are usually used for attributes. 

Worser example: `ida ~use_ida something` why do I need both `ida` and `use_ida`, if I'm calling to something with name `ida`, I definitely assume, that I'm going to use IDA, some what is the meaning of the `use_ida` option? (Of course I know, what you're doing, I'm talking from the user/reader perspective).